apps/shell/tash_command.c

```diff
@@ -213,7 +213,7 @@ static int tash_launch_cmdtask(TASH_CMD_CALLBACK cb, int argc, char **args)
 	}
 #endif
 
-	printf("Command will be launched with pri (%d), stack size(%d)\n", pri, stack_size);
+	shdbg("Command will be launched with pri (%d), stack size(%d)\n", pri, stack_size);
```

Comment:
1. shvdgb is correct because this is an information sentence.
2. If we change this to debug output, when we execute a command which is set to ASYNC, we can not see any result. If it has low priority or it is silrently working, in my opinion, I wonder it whether it is executed or not. 
Is it ok to you?

Reply:
1. Yes, it's an informational log message, should be changed to `shvdbg`
2. But this log message doesn't say whether the ASYNC command is executed or not. It's just an info by TASH about launching of command.

Reply:
2. Right. But we can know that is launched and will be executed. I don't care when it will be scheduled. So, when I execute something, I want to know whether executed or not.

Reply:
1. not changed. It has ```shdbg``` yet.

Reply:
2. Because I think you miss-understood what I mean, I left previous comment. If you think that is ok (changed to shvdbg), let's carry on yours.